index.md

```diff
@@ -33,7 +33,7 @@ _The corresponding 2.1 test fixture(s) is given like this, where appropriate: ..
 * [Embedding HTML in descriptive properties][0007] (64)
 * Rights statement(s) (7)
 * thumbnail algorithm / discussion
-* Book (simplest, > 1 canvas) (19)
+* [Multiple Related Images (Book, etc.)][0009] (19)
```

Comment:
Could you update the title to "Simple Manifest - Book"

Reply:
Done in #209